src/js/actions/index.js

```diff
@@ -293,6 +292,50 @@ export function fetchSummary() {
   }
 }
 
+// used to trigger csv download properly
+function detectIE() {
+  const ua = window.navigator.userAgent;
+
+  const msie = ua.indexOf('MSIE ');
+  if (msie > 0) {
```

Comment:
I'm a little hesitant of parsing the user agent, but I'll allow it if you turn the comparison operator to `>=` since this would fail if the matching string in a new ie version happened to start at position 0